packages/apollo-datasource-rest/src/connectors/memcached.ts

```diff
@@ -27,8 +29,17 @@ export default class MemcachedKeyValueCache implements KeyValueCache {
   ): Promise<void> {
     const { ttl, tags } = Object.assign({}, this.defaultSetOptions, options);
 
+    // get and incr version number
```

Comment:
One thing I'd like to avoid is paying the price for keeping a version (or versions) when a particular entry isn't using cache tags. In that case, I don't think we'll want to read (let alone increment) the version at all.